Cargo.toml

```diff
@@ -1,7 +1,29 @@
 [package]
-name = "yago4"
-version = "0.0.0"
+name = "yago"
+version = "4.0.0"
 edition = "2018"
+authors = ["Thomas Pellissier Tanon <thomas@pellissier-tanon.fr>",]
+description = "RDFS knowledge base, collection of facts based on Wikidata"
+repository = "https://github.com/yago-naga/yago4"
+documentation = "https://docs.rs/yago/"
+homepage = "https://yago-knowledge.org"
+readme = "README.md"
+keywords = ["knowledge", "base", "RDFS", ]
+categories = ["command-line-utilities", "science",]
+license = "GPL-3.0-or-later"
+
+include = [
+    "/src/**/*",
+    "Cargo.toml",
+    "LICENSE.md",
+    "README.md",
+    "rustfmt.toml",
+    "CHANGELOG.md",
+]
+
+
+[badges]
+maintenance = { status = "actively-developed" }
```

Comment:
I believe the state is more "Actively Maintained" now. No one is in charge of building new features anymore.

Reply:
```
# Maintenance: `status` is required. Available options are:
# - `actively-developed`: New features are being added and bugs are being fixed.
# - `passively-maintained`: There are no plans for new features, but the maintainer intends to
#   respond to issues that get filed.
```
But I guess bugs will be fixed? That's a copy of the definition from the Manifest from [here](https://doc.rust-lang.org/cargo/reference/manifest.html#the-badges-section)

Reply:
I see, so actively developed makes sense.